### PR TITLE
Repair register new user command

### PR DESCRIPTION
**Pull request by keliramu**

### CODE REVIEW

File: daemon/rpc_login.go [hunk 1]
```diff
@@ -135,14 +141,24 @@ func (r *RPC) loginCommon(customCB customCallbackType) (payload *pb.LoginRespons
 }
 
 // LoginOAuth2 is called when logging in with OAuth2.
-func (r *RPC) LoginOAuth2(in *pb.Empty, srv pb.Daemon_LoginOAuth2Server) error {
+func (r *RPC) LoginOAuth2(in *pb.LoginOAuth2Request, srv pb.Daemon_LoginOAuth2Server) error {
```
Comment (mariusSincovici) [inline daemon/rpc_login.go:144]: This is not sending the login success/fail event, is it sent from somewhere else?

File: protobuf/daemon/login.proto [hunk 2]
```diff
@@ -4,6 +4,21 @@ package pb;
 
 option go_package = "github.com/NordSecurity/nordvpn-linux/daemon/pb";
 
+enum LoginType {
+  LoginType_UNKNOWN = 0;
```
Comment (mariusSincovici) [inline protobuf/daemon/login.proto:8]: [small] can it ever be `UNKNOWN`?

Comment (keliramu) [reply protobuf/daemon/login.proto:8]: it's grpc common/recommended practice to have `None` value e.g. when creating variable of that type to not get assigned some valid value which is not intended.

File: daemon/rpc_login.go (hunk 1 above)
Comment (keliramu) [reply daemon/rpc_login.go:144]: Yet event duration has to be calculated correctly

**Review by mariusSincovici — APPROVED** [review]

**Review by devzbysiu — APPROVED** [review]

**Pull request merged**